src/SortPeopleByDateOfBirthTest.java

```diff
@@ -61,13 +63,44 @@ void should_ReturnPeopleSortedByDateOfBirth_When_SortPeopleInDescendingOrder(){
         LocalDate.parse("1978-01-12"),
         LocalDate.parse("1969-03-15")
 
-
     );
 
-    assertEquals(expectedDatesOfBirth,actualDatesOfBirth);
+    assertEquals(expectedDatesOfBirth, actualDatesOfBirth);
+
+  }
+
+  @Test
+  void shouldReturnTrue_When_SearchAndPrintPerson() {
+    //given
+    List<Person> people = new ArrayList<>();
+    people.add(new Person("John", LocalDate.parse("1969-03-15"), "+447984356766", "john@example.com"));
+    people.add(new Person("Jane", LocalDate.parse("1998-04-09"), "+447220512328", "jane@example.com"));
+    people.add(new Person("Harry", LocalDate.parse("1980-09-25"), "+447220012555", "harry@example.com"));
+    people.add(new Person("Anne", LocalDate.parse("1978-01-12"), "+447220012222", "anne@example.com"));
+    people.add(new Person("Jack", LocalDate.parse("1996-08-20"), "+447220012098", "jack@example.com"));
+    //when, then
 
+    assertTrue(SortPeopleByDateOfBirth.searchAndPrintPerson(people, "jane@example.com", "+447220512328"));
+    assertTrue(SortPeopleByDateOfBirth.searchAndPrintPerson(people, "dom@example.com", "+447220012555"));
+    assertTrue(SortPeopleByDateOfBirth.searchAndPrintPerson(people, "anne@example.com", "+447220012000"));
   }
 
+  @Test
+  void shouldReturnFalse_When_SearchAndPrintPerson() {
```

Comment:
I think the test name could be more explicit with it returning false IF an email address does not match.

Additionally you can add another expecting false test where the email and phone number does not match